requirements.txt

```diff
@@ -0,0 +1,2 @@
+lbt-ladybug==1.6.2
```

Comment:
```suggestion
lbt-ladybug==0.4.3
```

Reply:
Ah yes you're right my bad. I fixed this in 18af556b7cc242f351e429464a660ac56c54fc38 and also removed ladybug-geometry